src/Classes/ServiceAPI/MyRadio_Selector.php

```diff
@@ -196,11 +196,12 @@ public static function setStudio($studio)
             throw new MyRadioException('Selector Locked');
         } elseif ($response === 'ACK') {
             return [
```

Comment:
this block (now) appears to be pretty much identical to `getStatusAtTime` - deduplicate?